prediction_market_agent/agents/safe_guard_agent/safe_api_utils.py

```diff
@@ -74,12 +82,19 @@ def get_safe_queued_transactions(
         f"https://safe-client.safe.global/v1/chains/{RPCConfig().chain_id}/safes/{safe_address}/transactions/queued"
     ).json()
     response_parsed = TransactionResponse.model_validate(response)
-    transactions = [
-        check_not_none(item.transaction)
-        for item in response_parsed.results
-        if is_valued_transaction_result(item, response_parsed.results)
+    return [r.transaction for r in response_parsed.results if r.transaction is not None]
+
+
+def get_safe_queue_multisig(
```

Comment:
Comparet do the standard `get_safe_queue`, this will return only multi sig transactions (including the narrowed model as seen above). Because only these (I hope 😄 ) makes sense for the agent to check.